manifests/jetty.pp

```diff
@@ -110,6 +106,13 @@
     mode    => '0644',
     recurse => false,
     notify  => Class['shibboleth_idp::service'],
+  } ->
+
+  file { $idp_jetty_log_dir:
+    ensure => directory,
+    owner  => $shibboleth_idp::shib_user,
+    group  => $shibboleth_idp::shib_group,
+    mode   => '0644',
```

Comment:
The management and recursion will stop at the jetty_files, since we have a more specific file resource for them.